_opensearch/rest-api/document-apis/delete-by-query.md

```diff
@@ -59,6 +59,7 @@ scroll | Time | Amount of time the search context should be open.
 scroll_size | Integer | Size of the operation's scroll requests. Default is 1000.
 search_type | String | Whether OpenSearch should use global term and document frequencies calculating revelance scores. Valid choices are `query_then_fetch` and `dfs_query_then_fetch`. `query_then_fetch` scores documents using local term and document frequencies for the shard. It’s usually faster but less accurate. `dfs_query_then_fetch` scores documents using global term and document frequencies across all shards. It’s usually slower but more accurate. Default is `query_then_fetch`.
 search_timeout | Time | How long to wait until OpenSearch deems the request timed out. Default is no timeout.
+slices | String or Integer | How many slices to cut the operation into for faster processing. Available options are `auto`, which tells OpenSearch it should decide how many slices to divide into, or any number of shards. If you have a lot of shards in your index, set a lower number for better efficiency. Default is 1, which means the task should not be divided.
```

Comment:
Does "how many slices to divide into, or any number of shards" mean "how many slices to divide the operation into, for any number of shards"?

Reply:
Right, that's what I was shooting for. I've tweaked some language, which should (hopefully) clear things up.